layouts/default.vue

```diff
@@ -61,13 +61,16 @@
     text-align: left;
   }
 
-  .title {
+  h1 {
     font-family: 'Open Sans', Helvetica, Arial, sans-serif;
     display: block;
-    font-weight: 300;
-    font-size: 1.6rem;
-    line-height: 3.74rem;
+    font-weight: 300 !important;
+    font-size: 1.6rem !important;
+    line-height: 3.74rem !important;
```

Comment:
As @brampauwelyn recently suggested, we should try to avoid `!important` unless there's really no other way.

> The only thing that can override an !important tag is another !important tag. By using it one once, you potentially end up with a CSS file that is full of !important tags, which is not ideal.

https://uxengineer.com/css-specificity-avoid-important-css/

Seeing this change now, I wonder why you added them? Does that mean the `h1` tags never had these styles before either? Or is there a `h1` you added now in this PR that didn't get those styles?

Reply:
the heading styles was on the class .title, I think if we want to reuse headings it should be on the h1-tag, so it can be used also without the .title class. But the !important seems to be needed to override bootstrap defaults for headings

Reply:
unless I can override these by overriding global variables of bootstrap

Reply:
Are you sure? That seems strange. Bootstrap doesn't use a very specific selector, just `h1`. It doesn't use `!important` either. So it should be easy to override? Unless Bootstrap gets included _after_ our own styling of course 🤔 That would be a big problem in general down the line. So maybe double-check that first.

If that fails, you could try a more specific selector like `body h1` for example. Even if Bootstrap overrides the style later based on the `h1` selector, the `body h1` selector will have precedence because it's more specific. The article I linked also explains that concept.